src/js/common/schemaform/FormStartControls.jsx

```diff
@@ -53,8 +53,8 @@ class FormStartControls extends React.Component {
         <div>
           <ProgressButton
               onButtonClick={this.handleLoadForm}
-              buttonText="Resume previous application"
-              buttonClass="usa-button-primary"/>
+              buttonText="Continue with Application"
+              buttonClass="usa-button-primary title-case"/>
```

Comment:
Oh, I see, I think I misunderstood this initially. We're still title casing, we just don't want to do it the way text-transform does. I think that's ok, then, but I think we should choose something other than `title-case`. `no-text-transform` or something.